examples/hHX_set_persistent/hHX_set_persistent.ino

```diff
@@ -0,0 +1,138 @@
+
```

Comment:
need to add an .arduino-ci.yml file (see conversation)

Reply:
the example is now for arduino uno